src/app.jsx

```diff
@@ -63,16 +64,19 @@ class App extends React.Component {
       tableInfoWindow: {
         isHidden: true
       },
-      layers: []
+      layers: [],
+      timelineDate: new Date('2012-01-01')
     };
-
-    // this._setListeners();
   }
 
   _setListeners() {
     this.state.layersSpecCollection.on('change reset', () => {
       this.refs.Map.updateLayers();
     }).bind(this);
+
+    Backbone.Events.on('timeline:update', function(params) {
+      this.updateRouter(params);
+    } ,this)
   }
 
```

Comment:
We already call an app method from the timeline to update router. The only thing you need to update aside of this is the app state. (already on that function `updateTimelineDates(){}`)